test/modules/TestResultCoordinator/Reports/DirectMethod/DirectMethodReport.cs

```diff
@@ -64,7 +64,8 @@ class DirectMethodReport : TestResultReportBase
         // MismatchFailure is when the there is a result in ActualStore but no result in ExpectedStore. This should never happen.
         public ulong MismatchFailure { get; }
 
-        public override string Title => $"DirectMethod Report for [{this.ExpectedSource}] and [{this.ActualSource}] ({this.ResultType})";
+        public override string Title => this.ReceiverSource.HasValue ?
+            $"DirectMethod Report ({this.ResultType}) for [{this.SenderSource}] and [{this.ReceiverSource.OrDefault()}]" : $"DirectMethod Report ({this.ResultType}) for [{this.SenderSource}]";
```

Comment:
I already changed other reports to move result type to the end.
```suggestion
            $"DirectMethod Report for [{this.SenderSource}] and [{this.ReceiverSource.OrDefault()}] ({this.ResultType})" : $"DirectMethod Report for [{this.SenderSource}] ({this.ResultType})";
```